src/translations/en.json

```diff
@@ -2193,7 +2193,7 @@
             "glance": {
               "name": "Glance",
               "columns": "Columns",
-              "state_color": "Show State Color?",
+              "state_color": "Color icons based on state",
```

Comment:
all other options have question marks on end ( for example https://github.com/home-assistant/frontend/blob/dev/src/translations/en.json#L2249)
shouldn't it be added here for consistency?

Reply:
```suggestion
              "state_color": "Color icons based on state?",
```